.github/workflows/test.yml

```diff
@@ -2,7 +2,7 @@ name: "test"
 on:
   pull_request:
     paths-ignore:
-      - '*.md'
+      - '**.md'
```

Comment:
I always forget we need double asterisks to match a file change beyond the root level.